src/components/cc-addon-header/cc-addon-header.smart-matomo.js

```diff
@@ -0,0 +1,142 @@
+import { fakeString } from '../../lib/fake-strings.js';
+import { notify, notifyError } from '../../lib/notifications.js';
+import { defineSmartComponent } from '../../lib/smart/define-smart-component.js';
+import { generateDocsHref } from '../../lib/utils.js';
+import { i18n } from '../../translations/translation.js';
+import '../cc-smart-container/cc-smart-container.js';
+import { CcAddonHeaderClient } from './cc-addon-header.client.js';
+import './cc-addon-header.js';
+
+const DOCS_URL = generateDocsHref(`/addons/matomo`);
+const PROVIDER_ID = 'matomo';
+
+/**
+ * @typedef {import('./cc-addon-header.js').CcAddonHeader} CcAddonHeader
+ * @typedef {import('../../lib/smart/smart-component.types.js').OnContextUpdateArgs<CcAddonHeader>} OnContextUpdateArgs
+ */
+
+defineSmartComponent({
+  selector: 'cc-addon-header[smart-mode=matomo]',
+  params: {
+    apiConfig: { type: Object },
+    ownerId: { type: String },
+    addonId: { type: String },
+    logsUrlPattern: { type: String },
+    productStatus: { type: String, optional: true },
+  },
+
+  /** @param {OnContextUpdateArgs} args */
+  onContextUpdate({ context, updateComponent, onEvent, signal }) {
+    const { apiConfig, ownerId, addonId, productStatus } = context;
+    const api = new CcAddonHeaderClient({ apiConfig, ownerId, addonId, providerId: PROVIDER_ID, signal });
+    let logsUrl = '';
+
+    updateComponent('state', {
+      type: 'loading',
+      logsUrl: fakeString(15),
+      openLinks: [
+        {
+          url: fakeString(15),
+          name: fakeString(5),
+        },
+      ],
+      actions: {
+        restart: true,
+        rebuildAndRestart: true,
+      },
+      productStatus: fakeString(4),
```

Comment:
**nitpick:** since Matomo is not in beta or anything, we could remove the product status from the skeleton

Reply:
even better, I'm only realizing it now: we should not ever use fake strings for `productStatus`, we should just use the `productStatus` we get from `context`.
It's not something we load from the API anyway, it's given from the start by the console so we may as well use it right away.

This is something we should do for all add-ons, this way we only show a skeleton productStatus when needed.


```suggestion
      productStatus,
```